node/src/main/scala/coop/rchain/node/web/package.scala

```diff
@@ -1,6 +1,7 @@
 package coop.rchain.node
 
-import cats.effect.Concurrent
+import cats.implicits._
```

Comment:
Is _implicits_ really needed here or `cats.syntax.all._` is enough?

Reply:
I am curious how you can see that. Sometimes, I kind of in a mist of which I should import.